lib/adapters/JasmineAdapter.js

```diff
@@ -0,0 +1,66 @@
+import EventEmitter from "../eventemitter";
+import {Test, Suite} from "../data";
+
+/*global jasmine*/
+export default class JasmineAdapter extends EventEmitter {
+    constructor() {
+        super();
+        jasmine.getEnv().addReporter({
+            specDone: this.onSpecDone.bind(this),
+            specStarted: this.onSpecStarted.bind(this),
+            suiteStarted: this.onSuiteStarted.bind(this),
+            suiteDone: this.onSuiteDone.bind(this),
+            jasmineDone: this.onJasmineDone.bind(this)
+        });
+        // a stack of test arrays. The top element on the stack is the currently active suite
+        this.tests = [[]];
+        // a stack of suite arrays
+        this.suites = [[]];
+    }
+
+
+    onSpecStarted() {
+        this.startTime = new Date();
+    }
+
+    onSpecDone(details) {
+        var runtime = new Date() - this.startTime;
+        var skipped;
+        if (details.status === "pending") {
+            skipped = true;
+        } else {
+            skipped = false;
+        }
+        var errors = details.failedExpectations;
+
+        var test = new Test(
+            details.description,
+            details.fullName,
+            details.status,
```

Comment:
This looks like its wrong at least for `skipped`, where it now ends up as `pending` instead. Also makes me wonder if the other states are consistent with the ones from the QUnit adapter.